src/Build/Evaluation/LazyItemEvaluator.cs

```diff
@@ -151,15 +151,15 @@ public ItemData Clone(IItemFactory<I, I> itemFactory, ProjectItemElement initial
 
         private class MemoizedOperation : IItemOperation
         {
-            public IItemOperation Operation { get; }
+            public LazyItemOperation Operation { get; }
```

Comment:
I initially made the change when I wanted to batch different types of operations, since that would mean I could have a map from string to LazyItemOperation more easily. When I switched to only batching Update, I didn't see any reason to change it back. Is there one?